demo/compositions/useWorkerPoolQueueMock.ts

```diff
@@ -16,6 +17,22 @@ export function useWorkerPoolQueueMock(override?: Partial<WorkerPoolQueue>): Wor
   return workerPoolQueue
 }
 
-export function useWorkerPoolQueuesMock(count: number, override?: Partial<WorkerPoolQueue>): WorkerPoolQueue[] {
-  return repeat(count, () => useWorkerPoolQueueMock(override))
+export function useWorkerPoolQueuesMock(count: number, override?: () => Partial<WorkerPoolQueue>): WorkerPoolQueue[] {
+  const workerPool = useWorkerPoolMock()
```

Comment:
You probably want to just use the mocker directly here rather than the composition. That composition already inserts a worker pool into the mock database. And the useWorkerPoolMock might have more seeds in the future as well which aren't needed here.